inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/PdfAnnotationEditor.java

```diff
@@ -71,6 +73,11 @@ public PdfAnnotationEditor(String aId, IModel<AnnotatorState> aModel,
         super(aId, aModel, aActionHandler, aJCasProvider);
 
         add(new PdfAnnoPanel("vis", aModel, this));
+        try {
+            documentModel = new DocumentModel(getJCasProvider().get().getDocumentText());
+        } catch (IOException e) {
+            LOG.error("Unable to load data", e);
```

Comment:
Displaying an error to the user would be nice.

Reply:
Hm, how could I achieve this in the constructor?
I don't have a `AjaxRequestTarget` (right class name?).
Or should I set this kind of stuff on the first ajax request?

Reply:
It would seem to me this needs to be done outside the constructor - what happens when you switch to another document? I believe in this case, the editor is reused and only the model changes.

Reply:
I am certain that I switched documents in between and it worked, so I guess each time a document is opened a new instance is created.
But I will doublecheck later.

Reply:
Seems to work.
I also remembered that I moved content from `setPdfExtractFile` to the constructor as well in my local ligature branch, which is also working.
I could move it to a method `load` in the `PdfAnnotationEditor` which is called by the `PdfAnnoPanel` once the file is request via ajax.
Then I could pass the `AjaxRequestTarget`.

Reply:
@reckart Can I maybe postpone this to the next (ligature) PR?

Reply:
nevermind, I changed it